content/docs/concepts/multicluster-deployments/index.md

```diff
@@ -42,9 +42,11 @@ Istio control plane. There are two possible deployment approaches:
 
 1. Multiple synchronized Istio control planes that have replicated service and routing configurations.
```

Comment:
This wording implies that the control planes are synching and share config themselves. However, they are independent (not necessarily identical) control planes and operators can pick and choose what services they want to connect using gateways and service entries. There is nothing currently synchronized automatically. Compared to the shared control plane approach, there is less manual replication.